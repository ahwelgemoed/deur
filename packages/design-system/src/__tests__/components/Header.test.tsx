import { cleanup, render, screen } from '@testing-library/react-native';

import { Header } from '../..';

afterEach(cleanup);

it('renders the header component ', async () => {
  const title = 'Hello Main Title';
  render(<Header>{title}</Header>);
  await screen.findByTestId('header_text');
  expect(screen.getByText(title)).toBeTruthy();
  expect(screen.toJSON()).toMatchSnapshot();
});
