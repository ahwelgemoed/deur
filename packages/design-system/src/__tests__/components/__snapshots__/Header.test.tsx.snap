// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  1`] = `
<View
  testID="header_view"
>
  <Text
    testID="header_text"
  >
    Hello Main Title
  </Text>
</View>
`;
